paleontographica-abteilung-b-palaeobotany-palaeophytology.csl

```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Paleontographica Abteilung B - Palaeobotany - Palaeophytology</title>
+    <id>http://www.zotero.org/styles/paleontographica-abteilung-b-palaeobotany-palaeophytology</id>
+    <link href="http://www.zotero.org/styles/paleontographica-abteilung-b-palaeobotany-palaeophytology" rel="self"/>
+    <link href="http://www.zotero.org/styles/annalen-des-naturhistorischen-museums-in-wien" rel="template"/>
+    <link href="https://www.schweizerbart.de/journals/palb/instructions" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>2194-900X</issn>
+    <eissn>2509-839X</eissn>
+    <summary>Style for Annalen des Naturhistorischen Museums in Wien</summary>
+    <updated>2017-04-04T12:34:19+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds</multiple>
+      </term>
+      <term name="no date">undated</term>
+    </terms>
+  </locale>
+  <macro name="anon">
+    <choose>
+      <if variable="author editor translator" match="none">
+        <text term="anonymous" text-case="capitalize-first"/>
```

Comment:
and we're sure this isn't getting abbreviated? "Anonymous" seems very unusual compared to "Anon."